articles/flows/guides/device-auth/includes/request-tokens.md

```diff
@@ -0,0 +1,74 @@
+## Request Tokens
+
+While you are waiting for the user to authorize the device, begin polling the token URL to request an Access Token. Using the extracted polling interval (`interval`) from the previous step, you will need to `POST` to the [token URL](/api/authentication#device-auth) sending along the `device_code`.
+
+### Example POST to token URL
+
+```har
+{
+  "method": "POST",
+  "url": "https://${account.namespace}/oauth/token",
+  "headers": [
+    { "name": "Content-Type", "value": "application/x-www-form-urlencoded" }
+  ],
+  "postData": {
+    "text": "{\"grant_type\":\"urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Adevice_code\",\"device_code\": \"YOUR_DEVICE_CODE\", \"client_id\": \"${account.clientId}\" }"
+  }
+}
+```
+
+#### Parameters
+
+| Parameter Name  | Description |
+|-----------------|-------------|
+| `grant_type`    | Set this to "urn:ietf:params:oauth:grant-type:device_code". This is an extension grant type (as defined by Section 4.5 of [RFC6749](https://tools.ietf.org/html/rfc6749#section-4.5)). |
+| `device_code`   | The `device_code` retrieved in the previous step of this tutorial. |
+| `client_id`     | Your application's Client ID. You can find this value in your [Application Settings](${manage_url}/#/Applications/${account.clientId}/settings). |
+
+### Responses
+
+While you wait for the user to authorize the device, you may receive a few different `HTTP 400` responses:
```

Comment:
```suggestion
While you wait for the user to authorize the device, you may receive a few different `HTTP 4xx` responses:
```
Each error has more or less different 4xx error codes. Dismiss this if you knew and `HTTP 400` is to avoid confusion.

Reply:
I knew, but I think it's good to be more precise here. Accepting your suggestion.